linera-core/src/local_node.rs

```diff
@@ -401,7 +402,7 @@ where
         validators: &[RemoteNode<impl ValidatorNode>],
         chain_id: ChainId,
         target_next_block_height: BlockHeight,
-        notifications: &mut impl Extend<Notification>,
+        notifications: &impl NotificationSink<Notification>,
```

Comment:
It would be nice to `impl<T: NotificationSink> NotificationSink for &'_ T` so we don't have to be specific about the `&` here.

Reply:
But it makes a difference: In general, `&T` isn't `Send` anymore.

Reply:
It might even make sense to require that `Notifier: Send`.

Reply:
I did, didn't I?